Model/Sales/TrackTraceHolder.php

```diff
@@ -111,45 +120,53 @@ public function createTrackTraceFromShipment(Order\Shipment &$shipment)
      */
     public function convertDataFromMagentoToApi($magentoTrack, $options)
     {
-        $address = $magentoTrack->getShipment()->getShippingAddress();
+        $this->consignment = ConsignmentFactory::createByCarrierId(BpostConsignment::CARRIER_ID);
+
+        $address      = $magentoTrack->getShipment()->getShippingAddress();
         $checkoutData = $magentoTrack->getShipment()->getOrder()->getData('delivery_options');
-        $deliveryType = $this->getDeliveryTypeFromCheckout($checkoutData);
+        $deliveryType = $this->consignment->getDeliveryTypeFromCheckout($checkoutData);
+
         if ($options['package_type'] === 'default') {
             $packageType = self::$defaultOptions->getPackageType();
         } else {
-            $packageType = (int)$options['package_type'] ?: 1;
+            $packageType = (int) $options['package_type'] ?: 1;
         }
 
-        if ($address->getCountryId() != 'BE' && (int)$options['package_type'] == 2) {
+        if ($address->getCountryId() != 'NL' &&
+            ((int) $options['package_type'] == 2 || (int) $options['package_type'] == 4)) {
```

Comment:
graag contante gebruiken voor alle integers + voor die 'NL'

Reply:
Deze hele check kan weg omdat België alleen maar [package_type 1](https://myparcelbe.github.io/api/#package_type) heeft.

Denk dat dit is meegekomen met het compare met NL

Done